test/LayerNorm.test.ts

```diff
@@ -0,0 +1,164 @@
+import { expect, describe, it } from 'bun:test'
+import * as sm from '@shumai/shumai'
+import { areSameShape, expectArraysClose, expectThrows } from './utils'
+
+function stddev(vals: number[]): number {
```

Comment:
no need to change this (I figure it's useful to have an independent ref implemented), but there is `std` in shumai
https://facebookresearch.github.io/shumai/functions/std.html

Reply:
I cloned the branch locally and it looks as though our current ESLint/prettier rules don't fix the newline there, but I can probably identify the changes to config so this will be automatically cleaned up going forward!